src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.HandleObject.cs

```diff
@@ -52,6 +52,8 @@ private static void HandleStartObject(JsonSerializerOptions options, ref ReadSta
                     state.Current.ReturnValue = value;
                     state.Current.DetermineIfDictionaryCanBePopulated(state.Current.ReturnValue);
                 }
+
+                state.Current.CollectionPropertyInitialized = true;
```

Comment:
Was this a separate issue?

Reply:
No known/reported issues. Nothing else comes to mind beyond the current issue.